cluster.go

```diff
@@ -1760,12 +1775,9 @@ func (c *Cluster) multiCallWrapper(
 	var actualDests []peer.ID
 	var actualReplies []interface{}
 
-	authErr := fmt.Sprintf("client does not have permissions to this method, service name: %s, method name: %s", svcName, svcMethod)
-
 	for i, p := range dests {
-		if c.tc.Has(peer.IDB58Encode(p)) {
-			logger.Warningf("avoiding the request since it was unauthorized recently: peer %s, service name: %s, method name: %s", dests[i], svcName, svcMethod)
-			errs[i] = errors.New(authErr)
+		if c.unauthPeerTimeCache.Has(string(p)) {
+			errs[i] = fmt.Errorf("%s, peer: %s", authMsg, dests[i])
```

Comment:
the `isSkipError` will likely not work because you are not creating a skip error here.

`errs[i] = errUnauthPeerCache`